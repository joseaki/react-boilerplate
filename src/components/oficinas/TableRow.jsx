import React from "react";

const TableRow = ({ datos = {} }) => (
  <tr>
    <td>{datos.id}</td>
    <td>{datos.name}</td>
    <td>{datos.location}</td>
    <td>{datos.date}</td>
  </tr>
);
export default TableRow;
