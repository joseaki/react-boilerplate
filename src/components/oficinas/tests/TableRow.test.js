import React from "react";
import ReactDOM from "react-dom";
import TableRow from "../TableRow";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";

it("coponente renderiza sin error", () => {
  const div = document.createElement("tbody");
  ReactDOM.render(<TableRow></TableRow>, div);
});

it("matches snapshot", () => {
  const tree = renderer.create(<TableRow></TableRow>).toJSON();
  expect(tree).toMatchSnapshot();
});
