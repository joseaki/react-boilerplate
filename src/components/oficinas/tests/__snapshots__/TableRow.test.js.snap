// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<tr>
  <td />
  <td />
  <td />
  <td />
</tr>
`;
